sys/include/net/af.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_af  UNIX address families
+ * @ingroup     net
+ * @brief       Global UNIX address family definitions
+ * @{
+ *
+ * @file
+ * @brief   Global UNIX address family definitions
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef AF_H_
+#define AF_H_
+
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum {
+    AF_UNSPEC = 0,              /**< unspecified address family */
+#define AF_UNSPEC   AF_UNSPEC   /**< unspecified address family (as macro) */
+    AF_UNIX,                    /**< local to host (pipes, portals) address family. */
+#define AF_UNIX     AF_UNIX     /**< unspecified address family (as macro) */
+    AF_INET,                    /**< internetwork address family: UDP, TCP, etc. */
+#define AF_INET     AF_INET     /**< internetwork address family: UDP, TCP, etc. (as macro) */
+    AF_INET6,                   /**< internetwork address family with IPv6: UDP, TCP, etc. */
+#define AF_INET6    AF_INET6    /**< internetwork address family with IPv6: UDP, TCP, etc.
+                                 *    (as macro) */
+    AF_NUMOF,                   /**< maximum number of address families on this system */
+#define AF_NUMOF    AF_NUMOF    /**< maximum number of address families on this system (as macro) */
+};
```

Comment:
Are we ever gonna use this type? Usually `AF_` constants are `int` throughout the code and in POSIX, so de-anonymising this type would require casting everywhere it is used. In terms of code-beautification I like to prevent that.